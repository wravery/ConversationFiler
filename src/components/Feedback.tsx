import * as React from "react";

export class Feedback extends React.Component<undefined, undefined> {
    render() {
        return (<div className="feedback">
            <a href="https://beandotnet.azurewebsites.net/">about this app</a>
            &nbsp;
            <a href="mailto:wravery@hotmail.com?Subject=Auto%20Filer%20App%20for%20Outlook">send feedback</a>
        </div>);
    }
}