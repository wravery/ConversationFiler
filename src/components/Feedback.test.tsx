/// <reference path="../_testReferences.ts" />

import * as React from "react";
import * as renderer from "react-test-renderer";

import { Feedback } from "./Feedback";

test("Feedback (static content)", () => {
    const component = renderer.create(<Feedback />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
