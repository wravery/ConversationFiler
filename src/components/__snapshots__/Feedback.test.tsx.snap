// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feedback (static content) 1`] = `
<div
  className="feedback ms-font-s"
>
  <a
    href="https://beandotnet.azurewebsites.net/"
  >
    about this app
  </a>
  Â 
  <a
    href="mailto:wravery@hotmail.com?Subject=Conversation%20Filer%20v2.0%20App%20for%20Outlook"
  >
    send feedback
  </a>
</div>
`;
